pom.xml

```diff
@@ -944,8 +944,7 @@ under the License.
 
   <properties>
     <javaVersion>8</javaVersion>
-    <maven.compiler.source>${javaVersion}</maven.compiler.source>
-    <maven.compiler.target>${javaVersion}</maven.compiler.target>
+    <maven.compiler.release>${javaVersion}</maven.compiler.release>
```

Comment:
Should we left old properties and allow to m-compiler-p to use proper one?

What java version will be used by m-compiler-p when we have only maven.compiler.release and build on JDK 8?

Reply:
> Since version 3.13.0 of the Compiler Plugin together with the default javac compilerId you don't need conditional parametrisation of release. The release parameter will only be effective for Java 9 or above, otherwise the source and target will be passed to the compiler.